packages/devtools-reps/src/reps/error.js

```diff
@@ -20,7 +20,8 @@ const IGNORED_SOURCE_URLS = ["debugger eval code"];
 ErrorRep.propTypes = {
   object: PropTypes.object.isRequired,
   // @TODO Change this to Object.values when supported in Node's version of V8
-  mode: PropTypes.oneOf(Object.keys(MODE).map(key => MODE[key]))
+  mode: PropTypes.oneOf(Object.keys(MODE).map(key => MODE[key])),
+  renderStacktrace: PropTypes.func
```

Comment:
It would be great to have an explanatory comment for the new prop

```// Custom function used to render the stack trace (optional)
  renderStacktrace: PropTypes.func
```

Reply:
done ✅